pumba/src/main/java/org/arquillian/cube/q/pumba/PumbaStandaloneContainerHandler.java

```diff
@@ -0,0 +1,36 @@
+package org.arquillian.cube.q.pumba;
+
+import org.arquillian.cube.docker.impl.client.CubeDockerConfiguration;
+import org.arquillian.cube.docker.impl.client.config.DockerCompositions;
+import org.arquillian.cube.q.spi.StandaloneContainer;
+import org.arquillian.cube.q.spi.StandaloneManager;
+import org.jboss.arquillian.core.api.Instance;
+import org.jboss.arquillian.core.api.annotation.Inject;
+
+import java.util.Arrays;
+
+public class PumbaStandaloneContainerHandler implements StandaloneManager {
+
+    @Inject
+    Instance<CubeDockerConfiguration> cubeDockerConfigurationInstance;
+
+    @Override
+    public StandaloneContainer install(DockerCompositions dockerCompositions) {
```

Comment:
If we don't use `dockerCompositions` at all maybe it's time to rethink the interface?